recipes/superset/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "superset" %}
+{% set version = "0.28.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: d17192941353b93375ad593e1c812858bd36b25076481dca00d864fcd050e99f
+  - url: https://raw.githubusercontent.com/apache/incubator-superset/master/LICENSE.txt
+    sha256: 600710369a31face16d98c10682bda5d513fe1136f39f0c12f6f6cd9e2ccfe98
+  - patches:
+      - replace-scripts-with-console.patch
+
+build:
+  number: 0
+  script:
+    {% set console_script=os.sep.join(['superset', 'bin', 'superset']) %}
+    - mv {{ console_script }} {{ console_script}}.py  # [unix]
+    - "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or not py36]
+  entry_points:
+    - superset = superset.bin.superset:cli
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    # All dependency comments come from github
+    # https://raw.githubusercontent.com/apache/incubator-superset/0.28/requirements.txt
+    - bleach
+    - boto3 ==1.4.7
```

Comment:
This is an extremely old version of boto3.  That might cause some issues...

Reply:
@mariusvniekerk, that is the version that they had in their setup.py. I can relax the constraint, but I have no easy way to add test to meta.yaml and make sure other things did not break. I'll defer to your judgement. Should I change it to >=1.4.7,<2 and trust AWS' semver?